docs/assets/css/toggle-code-button.less

```diff
@@ -1,19 +1,15 @@
 // Button to show/hide code snippets and specs on component pages
-// When the `data-code-hidden` HTML attribute is present on the button element,
-// the button text shows "Show code" instead of "Hide code"
 
-.component-header {
-    position: relative;
-}
-
-#toggle-code-btn {
-    position: absolute;
-    right: 0;
-    top: 0;
-    .show-button { display: none; }
-    .hide-button { display: inline; }
-    &[data-code-hidden] {
-        .show-button { display: inline; }
-        .hide-button { display: none; }
+.a-toggle_code {
+    border-bottom: 1px solid @gray-40;
+    text-align: right;
+    margin: 0 -2rem;
+    padding: 0 2rem 1rem;
+    .a-btn {
+        text-transform: uppercase;
+        &.u-hidden {
```

Comment:
Why's this necessary? Isn't `u-hidden` already defined as `display:none`? Or does this not include cfpb-core?

Reply:
The `display: inline-block` on `.a-btn` overrides it. :/

<img width="581" alt="Screen Shot 2020-05-01 at 1 06 06 PM" src="https://user-images.githubusercontent.com/1060248/80824526-9e053a80-8bac-11ea-9019-100a928dc4db.png">

Reply:
wth that's not right